docker/centos-graal-maven-s2i/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# [Quarkus.io](http://quarkus.io) *native* S2I builder
+# Original Author: Michael Vorburger.ch
+
+# This part is heavily inspired by ../centos-graal/Dockerfile
+FROM centos:latest
+
+ARG GRAAL_VERSION
+ENV GRAAL_VERSION=${GRAAL_VERSION:-1.0.0-rc12}
+ARG MAVEN_VERSION=3.6.0
```

Comment:
Wondering if we can reuse one of the docker image Quarkus provides: https://github.com/quarkusio/quarkus/tree/master/docker

Avoiding having to maintain version is several places would be great.

Reply:
@cescoffier saw those and took inspiration from them.. the problem with doing e.g. a `FROM centos-graal-native-image` is that it makes it a bit harder to then set up CI/CD of this container - systems like Quay and Docker Hub except to run (exactly) 1 Dockerfile... so, at least for a first iteration, it seems simpler to me to keep it all together.

Reply:
Ok